### DISCUSSION TITLE
Error: The gRPC binary module was not installed

### DISCUSSION THREAD

**Discussion by NguyenAnhTuan1912** [vercel/community#782] [Help] [answered]
My app works fine on local, but doesn't when I deployed Vercel. I visit my app via Vercel's domain name, html. js and css work correctly. But when I navigate to another view then error occur. Function Log said "Error: The gRPC binary module was not installed. This may be fixed by running \"npm rebuild\"","Original error: Cannot find module '/var/task/node_modules/grpc/src/node/extension_binary/node-v93-linux-x64-glibc/grpc_node.node'"
I have just rebuild and install grpc but still not working.

**Comment by swarnava** [comment, marked as answer]
Hi,

A minimal reproduction repository is a stripped-down version of your repository. We need to be able to replicate the issue you are seeing to investigate, we can't at the moment. This could be due to factors such as your Build & Development settings, Environment Variables, etc.

Let us know if you have any further questions or concerns.
